caos.ansible_roles/roles/assert_service_status/tasks/assert-service-Linux.yaml

```diff
@@ -0,0 +1,20 @@
+---
+
+- name: Populate service facts
+  ansible.builtin.service_facts:
+
+- name: ensure service service is enabled
+  assert:
+    that: ansible_facts.services[item].status == 'enabled'
+  loop: "{{ services_enabled | flatten(levels=1) }}"
+
+- name: Wait for X seconds before checking if service is running
```

Comment:
I think you can put a variable in the name instead of X.

Reply:
👍 didn't know :)